pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/tablestate/TableStateUtils.java

```diff
@@ -38,6 +38,36 @@ public class TableStateUtils {
   private TableStateUtils() {
   }
 
+  /**
+   *
+   * TBD: reuse this in is all segments loaded below
+   */
+  public static List<String> getIdealStateOnlineSegments(HelixManager helixManager, String tableNameWithType) {
```

Comment:
let's rename this method to reflect that it is getting online segments for this instance

Reply:
Renamed